src/test/scala/uk/gov/hmrc/ct/ct600e/v2/E1032Spec.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.ct600e.v2
+
+import org.mockito.Mockito._
+import org.scalatest.mock.MockitoSugar
+import org.scalatest.{BeforeAndAfter, Matchers, WordSpec}
+import uk.gov.hmrc.cato.time.DateHelper
+import uk.gov.hmrc.ct.box.CtValidation
+import uk.gov.hmrc.ct.ct600e.v2.retriever.CT600EBoxRetriever
+import uk.gov.hmrc.ct.domain.ValidationConstants._
+
+class E1032Spec extends WordSpec with MockitoSugar with Matchers with BeforeAndAfter{
+
+  val boxRetriever = mock[CT600EBoxRetriever]
+  val NOW = DateHelper.now()
```

Comment:
I agree with you but unfortunately the way we use the DateHelper doesn't allow using fixed dates in tests because we're not able to stub the current time. At least I haven't found a way to do it.